### PR TITLE
Zone/Silverpine: Added spawns in phase 169 in Forsaken High Command

### PR DESCRIPTION
**Pull request by Nyr97**
* added spawns for missing creatures in phase 169 in Forsaken High Command.
* implemented Guts and Gore (questId 26995).
* implemented The Gilneas Liberation Front (questId 26989).
* implemented Iterating Upon Success (questId 26998).
* implemented (Serverside/Non-DB2) Summon Forsaken Warhorse (spellId 147066).

**Changes proposed:**

^  

**Issues addressed:**

None.

**Tests performed:**

It builds and it was tested in-game.

**Known issues and TODO list:** (add/remove lines as needed)

None.

### CODE REVIEW

File: src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp [hunk 1]
```diff
@@ -941,10 +943,488 @@ class spell_silverpine_forsaken_trooper_masterscript_high_command : public Spell
...
+    {
+        switch (player->PlayerTalkClass->GetGossipOptionAction(gossipListId))
+        {
+            case GOSSIP_ACTION_INFO_DEF:
+                Talk(TALK_MAGGOTHBREATH_ITERATING_UPON_SUCCESS, player);
+                player->CastSpell(player, SPELL_SUMMON_FORSAKEN_BAT, true);
+                break;
+
+            // Note: Blizzard messed up this OptionNpc. It sends GossipOptionNpc::None rather than GossipOptionNpc::TaxiNode,
+            // making it useless. To keep it blizzlike, we're just sending TaxiMenu forcefully as well.
```
Comment (offl) [inline src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp:1175]: This is weird

File: src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp [hunk 2]
```diff
@@ -941,10 +943,488 @@ class spell_silverpine_forsaken_trooper_masterscript_high_command : public Spell
...
+        me->m_spells[1] = 0;
+    }
+
+private:
+    EventMap _events;
+    ObjectGuid _playerGUID;
+};
+
+// Go Home - 83594
+class spell_silverpine_go_home : public SpellScript
```
Comment (offl) [inline src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp:1394]: You can just use SpellHit in AI, easier + on retail dummies generally aren't scripted but handled in AI's SpellHit

Comment (Nyr97) [reply src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp:1394]: Sure, I'll handle it that way on the next update.

File: src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp (hunk 1 above)
Comment (Nyr97) [reply src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp:1175]: I know, I was also baffled by this. I had to ask @meji46 as well for this, but the SQL data comes from their data directly.

File: src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp (hunk 2 above)
Comment (Nyr97) [reply src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp]: Feedback applied!

File: src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp [hunk 3]
```diff
@@ -941,10 +944,470 @@ class spell_silverpine_forsaken_trooper_masterscript_high_command : public Spell
...
+};
+
+// Deathstalker and Deathstalker Commander Belmont - 44789, 44790
+struct npc_silverpine_deathstalker : public ScriptedAI
+{
+    npc_silverpine_deathstalker(Creature* creature) : ScriptedAI(creature) { }
+
+    void JustAppeared() override
+    {
+        me->SetPowerType(POWER_ENERGY);
```
Comment (mdX7) [inline src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp:978]: looks kinda like this should be done in db, is the class properly set?

**Pull request merged**